elastic/kibana
# add x-elastic-internal-origin header to vector tile, glyphs, and fonts APIs

Closes https://github.com/elastic/kibana/issues/163311

To test
* create map with documents layer (that uses vector tile scaling, default).
* verify header is provided in request

<img width="500" alt="Screen Shot 2023-08-07 at 10 25 11 AM" src="https://github.com/elastic/kibana/assets/373691/0a057039-b8b5-4fba-8db2-aec91a2d519f">
